### ISSUE TITLE
Unable to recreate results

### ISSUE THREAD

**Issue by ntnshrav** [BruceWen120/medal#13]
Trained Electra model following instructions in the Github page. 
I also used the same train-test-val split as mentioned.
Trained Electra model on 2 GPUs (Multi-gpu setting) 2080ti's for 10 epochs. 
1. The training was for 10 epochs and took 5 whole days to train, Is that normal ? Can we make it faster?
2. There is no mention of how many epochs the model was trained to get the results furnished in paper. 
3. The validation accuracy is very low after training for 10 epochs 

    End of epoch 9
    Train Loss: 9.5633      Train Accuracy:0.0004
    Valid Loss: 9.5592      Valid Accuracy:0.0003
    ==================================================

It would be of great help if someone could help us with the above issues as soon as possible.

**Comment by BruceWen120** [comment]
Thanks for your interest in our work.

1. The dataset is quite large so it does take long to train. I don't remember the exact numbers, but 5 days for 10 epochs using ELECTRA fits my impression (assuming you're pre-training). There's definitely room for speeding up by optimizing the code and/or using a subset of the data, you are welcome to try.
2. Figure 5 and 6 in the paper show the performance vs training epochs. We didn't show that for mortality prediction, but it was similar to diagnosis prediction (not many epochs).
3. The model does not seem to be learning anything. Did you follow the hyper parameters in the paper?

**Comment by BruceWen120** [comment]
Close due to inactivity
